docs/proposals/configuration.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Configuration Files Design
+
+authors:
+  - "@kadisi"
+  - "@fisherxu"
+  
+approvers:
+  - "@kevin-wangzefeng"
+  - "@sids-b"
+
+creation-date: 2019-10-02
+
+---
+
+* [Configuration Files Design](#configuration-files-design)
+   * [Abstract](#abstract)
+   * [Goals](#goals)
+   * [Proposal](#proposal)
+      * [cloudcore](#cloudcore)
+         * [config struct](#config-struct)
+         * [default config file](#default-config-file)
+      * [edgecore](#edgecore)
+         * [config struct](#config-struct-1)
+         * [default config file](#default-config-file-1)
+
+# Configuration Files Design
+
+
+## Abstract  
+
+Currently, cloudcore/edgecore/edgesite\`s configuration files are in the conf directory at the same level and have 3 configuration files, it is difficult to maintain and extend.
```

Comment:
I guess the problem you want to address is not three configuration file we have, but drop those configuration files to the system directory rather than in the source.

Reply:
1 Tmerges three yaml files into one   2  reads the configuration file like k8s

Reply:
okay, I see two "default config file" in this proposal, it's better to tell they are only two different parts from one final config file.

Reply:
ok